Extensions/News/Compiler.js

```diff
@@ -0,0 +1,55 @@
+class News {
+    
+    /**
+     * 
+     * @param {HTMLElement} section 
+     */
+    constructor(section) {
+        this.section = section;
+        this.heading = null;
+        this.body = null;
+
+        this._parseHeading(this.section, (heading) => {
+            if (!heading) throw new Error("Nullptr found at [this._parseHeading] in class.News ctor<...>");
```

Comment:
does this pass lint rules? I thought we have a maximum line length of 80 characters

Reply:
We didn't agree yet on whether to use ESLint, and if so then how to use it. Therefore, I didn't run any of the code through ESLint. We can talk about this tmrw.